skore/src/skore/sklearn/_cross_validation/report.py

```diff
@@ -198,31 +201,55 @@ def _fit_estimator_reports(self) -> list[EstimatorReport]:
         )
 
         estimator_reports = []
-        try:
-            for report in generator:
-                estimator_reports.append(report)
-                progress.update(task, advance=1, refresh=True)
-        except (Exception, KeyboardInterrupt) as e:
-            from skore import console  # avoid circular import
+        for report in generator:
+            estimator_reports.append(report)
+            progress.update(task, advance=1, refresh=True)
+
+        warn_msg = None
+        if all(
+            isinstance(report, (KeyboardInterrupt, Exception))
+            for report in estimator_reports
+        ):
```

Comment:
Nitpick, in case in the future we change the behaviour of `_generate_estimator_report`: 

```suggestion
        if not any (
            isinstance(report, EstimatorReport)
            for report in estimator_reports
        ):
```